src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.cc

```diff
@@ -107,7 +107,17 @@ BtreeLBAManager::get_mapping(
   Transaction &t,
   laddr_t offset)
 {
-  ceph_abort("not implemented");
+  logger().debug("BtreeLBAManager::get_mapping: {}", offset);
+  return get_root(t
+  ).safe_then([this, &t, offset](auto extent) {
+    return extent->lookup_pin(get_context(t), offset
+    ).safe_then([extent] (auto ret) { return ret; });
```

Comment:
I'd suggest flatten this continuation chain, i.e., to promote the continuation above to the outer chain.

Reply:
why is `extent` captured?

Reply:
Hmm, seems no reason to keep the `extent` reference as `lookup_pin()` already handles it well internally.

Reply:
fixed.